sqlg-core/src/main/java/org/umlg/sqlg/structure/SqlgEdge.java

```diff
@@ -35,6 +35,8 @@ public SqlgEdge(SqlgGraph sqlgGraph, Long id, String schema, String table, SqlgV
         super(sqlgGraph, id, schema, table);
         this.inVertex = inVertex;
         this.outVertex = outVertex;
+        properties.clear();
```

Comment:
Not understanding this change. Does the following SqlgEdge.insertEdge not already add the properties?